ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
With one instance of the code pulled in, one deployment of Whisk is going to be there. As such, it should be part of global deployment, and it doesn't buy us anything moving the checks to under environment.